src/lang/Messages.properties

```diff
@@ -1094,6 +1094,8 @@ enc2.label.sha1Hash                    = SHA1 Hash
 enc2.label.text                        = Text to be encoded/decoded/hashed
 enc2.label.urlDec                      = URL Decode
 enc2.label.urlEnc                      = URL Encode
+enc2.label.unicode.escapedText         = EscapedText
```

Comment:
This (and following) should be two words? @kingthorin?

Reply:
Agreed.